jx_sqlite/aggs_table.py

```diff
@@ -377,7 +377,16 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("JSON_GROUP_ARRAY(DISTINCT(" + sql + "))")
+                        query_tables = tables.nest
+                        if details.nested_path not in query_tables:
+                            p = details.nested_path
+                            alias = nest_to_alias[p]
+                            from_sql += (
+                                "\nLEFT JOIN\n (SELECT " + PARENT + ", " + sql + " \nFROM " +
+                                quote_table(concat_field(base_table, p)) + ") " + alias +
```

Comment:
You can not use `alias`; choose a name that represents this select clause.  The `alias` could be in use when performing aggregations over an origin that is not the fact table.